jest.config.cjs

```diff
@@ -9,6 +9,7 @@ const externalNodeModules = [
   "@reactioncommerce/api-plugin-accounts",
   "@reactioncommerce/api-plugin-address-validation",
   "@reactioncommerce/api-plugin-address-validation-test",
+  "@reactioncommerce/api-plugin-i18n",
```

Comment:
@kieckhafer should I also move lines 7+8 under the `api-plugin-i18n` or they are left there for a reason?

Reply:
No reason, I updated them.